src/appendices/git/workflows.rst

```diff
@@ -0,0 +1,170 @@
+
+This workflow is similar to the :ref:`single-dev-branch` workflow, only now you must be mindful to merge in changes made by others.
+
+#. Make changes to your code
+#. Check status: ``git status``
+#. Stage changes: ``git add .``
+#. Commit changes: ``git commit -m "Added feature X"``
+#. Pull changes made by other: ``git pull origin master``
+#. Merge changes as necessary
+#. Push changes: ``git push origin master``
+
+.. _single-feature:
+
+Single Developer / Feature Branches
+-----------------------------------
+
+.. index:: 
+   single: Git; branch
+   single: Git; feature branch
+
+As you become more comfortable with Git and begin to build larger projects on your own, you'll want to use **feature branches**. This is a branch created in order to *isolate* changes related to a *single* feature or bug. Professional developers use them for a few reasons:
+
+- They can work on multiple new features at the same time, while keeping the work separated.
+- If a bug is found, they can commit changes to their feature branch and switch to the branch containing the bug.
+- Working in feature branches allows code to be easily reviewed by other developers before it is merged.
+
+.. admonition:: Fun Fact
+
+   Some development teams will *only* work in feature branches. On these teams, developers are NOT allowed to make changes directly to the ``master`` branch. They must work in feature branches and create pull requests into ``master`` to add those changes to the main code base.
+
+In the workflow below, we use ``main`` to refer to the branch from which the feature branch is created (often ``master``) and ``feature`` to refer to the feature branch. Remember that you should use descriptive names for your branches.
+
+#. Create a new branch. While in ``main`` create and move to the feature branch: ``git checkout -b feature``.
+#. Follow the :ref:`single-dev-branch` workflow until work on the feature is complete.
+#. After all changes in ``feature`` have been committed and pushed, move back to ``main``: ``git checkout main``.
+#. Merge your feature branch back into ``main``, resolving any conflicts: ``git merge feature``
+
+.. _team-feature:
+
+Team / Feature Branches
+-----------------------
+
+The workflow for a team of developers using feature branches combines the :ref:`team-single` and :ref:`single-feature` workflows. 
+
+In the workflow below, we use ``main`` to refer to the branch from which the feature branch is created (often ``master``) and ``feature`` to refer to the feature branch. Remember that you should use descriptive names for your branches.
+
+#. Create a new branch. While in ``main`` create and move to the feature branch: ``git checkout -b feature``.
+#. OR if contributing to a branch made by a team member, fetch and checkout their existing branch: ``git fetch origin`` then ``git checkout feature``
```

Comment:
Items 1 & 2 should be combined into the same bullet point. They are both the first step in the workflow, so they should be represented as a single either/or option.